site/docs/get-started/OneClickDemo.md

```diff
@@ -0,0 +1,45 @@
+# One Click Demo Guide
+
+this guide contains a script that you can run with a single command to see a demo of Fybrik in action.  
+
+this demo demonstrates the following sequence -  
```

Comment:
**The** demo demonstrates the following sequence**:**
0. A kind Kubernetes cluster is installed and Fybrik and all its dependencies are deployed to the cluster

Reply:
Fixed